File: data/deepseek_coder_base_6.7B/readable_format_HPEC/fortran/file109.f90
program DRB153_missinglock2_orig_gpu_yes
    use omp_lib
    implicit none

    integer :: var, i
    var = 0

    !$omp target map(tofrom:var) device(0)
    !$omp teams num_teams(1)
        !$omp distribute parallel do
        do i = 1, 100
            var = var + 1
        end do
        !$omp end distribute parallel do
    !$omp end teams
    !$omp end target

    print*, var
end program

File: data/CodeLlama/readable_format_HPEC/cpp/file109.cpp
#include <iostream>
#include <omp.h>

int main() {
    int var = 0;

    #pragma omp target map(tofrom:var) device(0)
    #pragma omp teams num_teams(1)
    {
        #pragma omp distribute parallel for
        for (int i = 1; i <= 100; ++i) {
            var = var + 1;
        }
    }

    std::cout << var << std::endl;

    return 0;
}
